pages/common/xmllint.md

```diff
@@ -0,0 +1,32 @@
+# xmllint
+
+> XML parser.
+> Also useful for detecting errors in XML code.
+
+- Return all nodes named {{foo}}:
```

Comment:
No need to wrap foo with tokens in the descriptions. You can just put it in quotes.